src/javascript/JContent/CompareDialog/FrameManager/FrameManager.jsx

```diff
@@ -65,7 +65,7 @@ const FrameManager = () => {
     // Load frames
     useEffect(() => {
         if (leftFrame.current && data?.jcr?.result?.renderUrlEdit) {
-            leftFrame.current.contentWindow.location.href = resolveUrlForLiveOrPreview(data.jcr.result.renderUrlEdit, false);
+            leftFrame.current.contentWindow.location.href = resolveUrlForLiveOrPreview(data.jcr.result.renderUrlEdit, false, null, true);
```

Comment:
I would not have modified the resolveUrlForLiveOrPreview function. 
I would just have added hidePersonaPanel=true at the end url like it's done in the customizedPreviewApp.jsx file
Example: resolveUrlForLiveOrPreview(renderUrl)}${urlParams}`
Here the modification would be:
 leftFrame.current.contentWindow.location.href = `${resolveUrlForLiveOrPreview(renderUrl)}?hidePersonaPanel=true`
And it would be nice to add a comment to explain that the hidePersonaPanel is related to jExperience and that modification is necessary to hide the persona panel in edit in the comparison page